Src/static/styles-public.css

```
@@ -421,3 +421,120 @@ footer {
   background-color: rgba(255, 255, 255, 0.07);
   color: var(--bs-body-color);
 }
+
+/* ===========================================
+   DARK THEME — MATCHING AUTHENTICATED PAGE
+   =========================================== */
+
+body {
+  font-family: "Inter", "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)